src/main/java/marquez/api/models/DatasourceResponse.java

```diff
@@ -0,0 +1,21 @@
+package marquez.api.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class DatasourceResponse {
+
+  @JsonProperty("name")
+  String name;
+
+  @JsonProperty("createdAt")
```

Comment:
We can remove `@JsonProperty` if the name remains unchanged. We also no longer need to annotate fields: https://github.com/MarquezProject/marquez/pull/342

Reply:
Nice!

Reply:
Please see PRs #394, #387, and #384, where this is fixed.